shared/src/test/java/org/eclipse/steady/shared/model/IExemptionTest.java

```diff
@@ -338,7 +338,7 @@ public void testCreatePackageUrl() {
     } catch (MalformedPackageURLException e) {
       assertTrue(true);
       assertEquals(
-          "Package URLs of type [maven] require a valid namespace and name: [" + purl + "]",
+          "The PackageURL specified is invalid. Maven requires both a namespace and name.",
```

Comment:
Why not to print anylonger the purl that raised the exception?

Reply:
This err message comes from the library, not from Steady, and it changed with the updated version